main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+)
+
+type Page struct {
+	Title string
+	Body []byte
+}
+
+func (p *Page) save() error {
+	filename := p.Title + ".txt"
+	return ioutil.WriteFile(filename, p.Body, 0600)
+}
+
+func loadPage(title string) (*Page, error) {
+	filename := title + ".txt"
+	body, err :=ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+	return &Page{Title: title, Body: body}, nil
+}
+
+func main() {
+	p1 := &Page{Title: "TestPage", Body: []byte("This is a sample Page.")}
+	p1.save()
+	p2, _ := loadPage("TestPage")
```

Comment:
Never skip errors in real code. you should always check returned error for a 'nil' and return function with error in there is something in. Because there could be some problems but you'll never know what exactly is wrong.
So code should look like this:
`p2, err := loadPage("TestPage")`
`   if err != nil { `
`   fmt.Println(err) // here you just print error content`
`   os.Exist(1) // here you exiting program with code 1, wich can be used by user to 
 identify that program exit whith error`
`}`